tests/functional/spec/percentage_decimal.spec.js

```diff
@@ -0,0 +1,15 @@
+import PercentagePage from "../generated_pages/percentage/block.page.js";
+import PercentageDecimalPage from "../generated_pages/percentage/block-decimal.page.js";
+import SubmitPage from "../generated_pages/percentage/submit.page.js";
+
+describe("Percentage", () => {
+  it("Given a percentage decimal answer, when I return to edit the value it does not amend the decimal places", () => {
```

Comment:
Percentage isn't important. So I would update the test to Decimal places

`Given an answer allows 3 decimal places, When I enter a value to 3 decimal places and return to edit the value, Then the answer should be displayed with 3 decimal places`


You will also need another one.


`Given an answer allows 3 decimal places, When I enter a value to 1 decimal place and return to edit the value, Then the answer should be displayed with 3 decimal places`

So enter `3.3`, on return it should be `3.300`.

Reply:
Test refactored.